app/instance-initializers/browser/head.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+export function initialize(instance) {
+  // clear fast booted head (if any)
+  Ember.$('meta[name="ember-cli-head-start"]')
+    .nextUntil('meta[name="ember-cli-head-end"] ~')
+    .addBack()
+    .remove();
+  const container = instance.lookup ? instance : instance.container;
```

Comment:
Should be able to remove this conditional, fastboot only supports ember 2.3.0+ so instance.lookup should always be present.